### ISSUE TITLE
twisted/python/zippath.py makes use of implements()

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7158]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7158|
|Type|enhancement|
|Created|2014-04-15 19:56:14Z|
|Branch|https://github.com/twisted/twisted/tree/zippath-implements-7158|

@implementer should be used instead.

<details><summary>Searchable metadata</summary>

```
trac-id__7158 7158
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_zippath_implements_7158 branches/zippath-implements-7158
branch_author__hynek hynek
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1397591774000000 1397591774000000
changetime__1474198049187875 1474198049187875
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex commented|
|-|-|

This is resolved by https://github.com/twisted/twisted/pull/15

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_2"></a>|@hynek commented|
|-|-|

(In [42282]) Branching to zippath-implements-7158.

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_3"></a>|@hynek set status to `closed`|
|-|-|

(In [42287]) Merge zippath-implements-7158

Author: Alex
Reviewer: hynek
Fixes: [#7158](https://github.com/twisted/twisted/issues/7158)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
